modules/local/pretext_graph.nf

```diff
@@ -4,19 +4,12 @@ process PRETEXT_GRAPH {
 
     container "quay.io/sanger-tol/pretext:0.0.8-yy5-c1"
 
-    // Exit if running this module with -profile conda / -profile mamba
```

Comment:
No conda bit removed but no conda option available in module? Should put back inside at the start of the script block (seems to be current best practice for location)

Reply:
Readded to this and another 2 modules.
Reformatted the container section for consistency.